apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/api/class-global-styles-info-rest-api.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * WPCOM Global Styles Info API.
+ *
+ * @package full-site-editing-plugin
+ */
+
+/**
+ * This class contains the necessary endpoints to interact with global styles outside the editor context.
+ */
+class Global_Styles_Info_Rest_API extends WP_REST_Controller {
+
+	/**
+	 * Class constructor
+	 */
+	public function __construct() {
+		$this->namespace                       = 'wpcom/v2';
+		$this->rest_base                       = 'global-styles-info';
```

Comment:
I've changed it to `global-styles/status` since it does make more sense from a semantic point of view. The only problem I see is that we aren't being consistent on path naming since we have some endpoints that don't follow this pattern within the global-styles domain.